lib/src/participant/remote.dart

```diff
@@ -42,6 +43,17 @@ class RemoteParticipant extends Participant<RemoteTrackPublication> {
     updateFromInfo(info);
   }
 
+  @override
+  EncryptionType get encryptionType {
```

Comment:
the implementation appears to be the same between local and remote. Could this be defined in participant.dart?